pandas/core/common.py

```diff
@@ -1857,23 +1857,84 @@ def _maybe_box_datetimelike(value):
 
 _values_from_object = lib.values_from_object
 
-
-def _possibly_convert_objects(values,
-                              datetime=True,
-                              numeric=True,
-                              timedelta=True,
-                              coerce=False,
+# TODO: Remove in 0.18 or 2017, which ever is sooner
+def _possibly_convert_objects(values, convert_dates=True,
```

Comment:
I'll just note that there is some value in keeping this function intact in `core._common`. Xray was using this private function internally, with a broken guard against it changing (see xray/xray#569 for details). Obviously, that was our fault and we've fixed it now.